### PR TITLE
[add] POST & DELETE routes for article without spec

### PR DESCRIPTION
**Pull request by Prajwalprakash3722**
- this PR introduces 2 new routes for article controller
- POST & DELETE

PS: is PUT request also needed?

- these features are not tested, test cases will be shortly written and pushed

### CODE REVIEW

**Comment by chrisvrose** [comment]
Please inspect #13  and #14 
Should be good examples. They're to be merged into this branch

**Comment by chrisvrose** [comment]
Now, I think only few improvements on post would be good

**Comment by Prajwalprakash3722** [comment]
> Now, I think only few improvements on post would be good

do you mean tests?

**Comment by Prajwalprakash3722** [comment]
What's this PR waiting for?

**Comment by chrisvrose** [comment]
I was gonna ask you xD
Will review then

**Comment by chrisvrose** [comment]
I was gonna ask you xD
Will review then

File: src/main/java/org/ieeervce/api/siterearnouveau/service/ArticleService.java [hunk 1]
```diff
@@ -19,4 +19,23 @@ public List<Article> list() {
     public Article getArticleById(Integer id) {
         return articlesRepository.findById(id).orElse(null);
     }
+
+    public Boolean saveArticle(Article article) {
```
[the remark this answers is not in the document]
Comment (chrisvrose) [reply src/main/java/org/ieeervce/api/siterearnouveau/service/ArticleService.java:23]: Please check if this part can be updated. Its should be useful.

[the remark this answers is not in the document]
Comment (Prajwalprakash3722) [reply src/main/java/org/ieeervce/api/siterearnouveau/service/ArticleService.java:23]: alright

File: src/main/java/org/ieeervce/api/siterearnouveau/controller/ArticlesController.java [hunk 2]
```diff
@@ -28,4 +30,17 @@ ResultsDTO<Article> getArticleById(@PathVariable("id") Integer id) {
         var article = articleService.getArticleById(id);
         return new ResultsDTO<>(article);
     }
+
+    @PostMapping()
+    ResultsDTO<Boolean> saveArticle(Article article) {
```
[the remark this answers is not in the document]
Comment (chrisvrose) [reply src/main/java/org/ieeervce/api/siterearnouveau/controller/ArticlesController.java]: Do you need help in this pr?
I can work on uts if you want

**Comment by Prajwalprakash3722** [comment]
Feel free to do

Prajwal P,
3rd Year Computer Science Student,
RVCE, Bengaluru-56
prajwalprakash3722.github.io/

**Comment by chrisvrose** [comment]
Cleans up all TODOs. Let's wait for CI.

**Review by chrisvrose — APPROVED** [review]
LGTM.

Delete API may need further thought, but let's carry on with this.
Thanks

**Pull request merged**